janusgraph-core/src/main/java/org/janusgraph/util/stats/MetricManager.java

```diff
@@ -200,7 +200,11 @@ public synchronized void removeJmxReporter() {
      * Create a {@link Slf4jReporter} attached to the JanusGraph Metrics registry.
      * <p>
      * If {@code loggerName} is null, or if it is non-null but
+<<<<<<< HEAD
      * {@link LoggerFactory#getLogger(String)} returns null, then Metrics's
+=======
+     * {@link LoggerFactory#getLogger(Class loggerName)} returns null, then Metrics's
+>>>>>>> 0.2
```

Comment:
missed a merge here

Reply:
Doh. It's been fixed now. I did a recursive grep on '>>>>>>>', '^=======$', and '<<<<<<<' to make sure that was the only one.